### PR TITLE
create new prediction_types for mean and median

### PR DESCRIPTION
**Pull request by matthewcornell**
closes [create new prediction_types for mean and median #367]. removes the beta feature to download the Zoltar database for local use

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -16,4 +16,4 @@ django = "*"
 [dev-packages]
 
 [requires]
-python_version = "3.6"
+python_version = "3.9"
```
Comment (bsweger) [inline Pipfile:19]: 🎉

**Review by bsweger — COMMENTED** [review]
My favorite kind of PR: 

image

The point --> mean/media changes look good

I started leaving a few inline comments, but I realized I could better summarize as:

- Can you confirm that the deleted files in `cli`, `forecast_app`, `forecast_repo`, and `settings` are all related to the beta download field that we're moving (it wasn't clear from the names)
- It wasn't in the PR, but might be a good time to upgrade the `actions/setup-python` version in the GitHub workflow (v5 is the current version). Additionally, if you add a `.python-version` file to the root of the repo, you don't need to explicitly specify the python version in the action itself ([more info](https://github.com/actions/setup-python?tab=readme-ov-file#basic-usage))

**Comment by matthewcornell** [comment]
Thanks very much for the comments, @bsweger . I've incorporated them in the new commit.

**Comment by matthewcornell** [comment]
@bsweger regarding your comment:
> Can you confirm that the deleted files in cli, forecast_app, forecast_repo, and settings are all related to the beta download field that we're moving (it wasn't clear from the names)

Yes, very good question. I'll break it down just to make sure it's clear. (Note the "OOPS")

```
cli
  bulk_data_load.py: cli app that creates and loads an sqlite database
  bulk_data_load_schema.sql: schema used by ""
  bulk_data_query.py: cli app to query forecasts in the sqlite database
  print_project_info.py: cli app to show info on projects in the database
  validate_quantile_csv_file.py: OOPS! this should not have been deleted. restored in the second commit above
forecast_app: contains Django models and support files copied from zoltar. required by forecast query feature
  models
    __init__.py
    forecast.py
    forecast_model.py
    prediction_data.py
    prediction_element.py
    project.py
    target.py
  __init__.py
  apps.py
forecast_repo: Ditto (Django support)
  settings
    base.py
    local_sqlite3.py
  __init__.py
utils: functionality called by cli apps
  project_queries.py
  project_truth.py
```

**Pull request merged**